### PR TITLE
Web3 Hackathon event website for the capstone project

### PR DESCRIPTION
**Pull request by V-Blaze**
Below are the major features and changes in this feature branch: 🥇 

- I create the 2 pages, the home page and the about page.
-  I adapt the 2 pages to desktop using media
- Implement the user interactions: link, mobile menu.
- Implement the section "Featured speakers" with dynamic HTML using JavaScript.

I added a link to a loom video about the project and my favorite part of the project in the ReadMe file. but here is also the link for easy access [Link to loom video](https://www.loom.com/share/b7948a3e23e64e81a8f0b631c0484b4b)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                </div>
+            <ul class="menu">
+                <li><a href="./about.html" class="nav-link">About</a></li>
+                <li><a href="./#program" class="nav-link">Program</a></li>
+                <li><a href="" class="nav-link">Join</a></li>
+                <li><a href="" class="nav-link">Sponsor</a></li>
+                <li><a href="" class="nav-link">News</a></li>
+                <li><a href="" class="nav-link">CC Campaign</a></li>
+            </ul>
+        </nav>
```
Comment (ha-manel) [inline index.html]: - [ ] for better semantics, it's better to wrap this section in `header` tag instead of `section` 👍

File: about.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                    <span class="bar"></span>
+                </div>
+            <ul class="menu">
+                <li><a href="./index.html" class="nav-link">Home</a></li>
+                <li><a href="./#program" class="nav-link">Program</a></li>
+                <li><a href="" class="nav-link">Join</a></li>
+                <li><a href="" class="nav-link">Sponsor</a></li>
+                <li><a href="" class="nav-link">News</a></li>
+                <li><a href="" class="nav-link">CC Campaign</a></li>
+        </nav>
```
Comment (ha-manel) [inline about.html]: - [ ] for better semantics, it's better to wrap this section in `header` tag instead of `section` 👍

File: about.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            Partners
+        </h2>
+        <hr>
+        <div class="partner-logos">
+            <img src="./assets/images/FintechSurge-Logo_new.png" alt="FintechSurge Logo">
+            <img src="./assets/images/Gitex-logo-2.png" alt="Gitex logo">
+            <img src="./assets/images/NORTH-STAR-DUBAI-LOGO-New-2.png" alt="NORTH STAR DUBAI LOGO New">
+        </div>
+
+    </section>
```
Comment (ha-manel) [inline about.html]: - [ ] Kindly try to follow design guidelines as much as possible. For the partners section the background color should be `#414246` and without `border-radius` for the top corners 👍 
## Your design:
image

## Guidelines:
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+                <div class="event-1">
+                    <div class="event-img-overlay">
+                        <!-- <img src="" alt="Past Event 1"> -->
+                        <span class="event-year"> 2021</span>
+                        <span class="event-detail"> Web3 Hackathon 2021 Lagos-Nigeria</span>
+                    </div>
+                </div>
+                <div class="event-2">
+                    <div class="event-img-overlay">
+                        <!-- <img src="" alt="Past event 2"> -->
```
Comment (ha-manel) [inline about.html]: - [ ] Kindly remove commented pieces of code for better readability and for a more professional code 💯

File: about.html (hunk 3 above)
Comment (ha-manel) [inline about.html]: - [ ] For the `about` page, the partners section is hidden in desktop and shown in mobile. Kindly adjust your website accordingly 👍  

## Your design:
image

## Guidelines:
image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+    <footer class="footer-section" id="footer">
+        <div class="mobile-footer">
+            <span>
+                <a href="/" class="footer-logo">
+                    <img src="./assets/images/Web3-Hackathon.png" alt="web3 Hackathon Logo">
+                </a>
+            </span>
+            <span>CopyRight of V-BLAZE 2022. All RIght Reserved</span>
+        </div>
+    </footer>
```
Comment (ha-manel) [inline about.html]: - [ ] For the `about` page, the footer in desktop is dark and in mobile is light, kindly make the necessary changes 👍 

## Your design:
image

## Guidelines:
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+    <section class="partner-footer-mobile">
+        <h2 class="partner-title">
+            Partners
+        </h2>
+        <hr>
+        <div class="partner-logos">
+            <img src="./assets/images/FintechSurge-Logo_new.png" alt="FintechSurge Logo">
+            <img src="./assets/images/Gitex-logo-2.png" alt="Gitex logo">
+            <img src="./assets/images/NORTH-STAR-DUBAI-LOGO-New-2.png" alt="NORTH STAR DUBAI LOGO New">
+        </div>
```
Comment (ha-manel) [inline index.html:145]: - [ ] In mobile view, these 2 sections are overlapping, kindly fix this issue 👍 

image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @V-Blaze 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
- [ ] Kindly deploy your website using `Github Pages` as required in the project specifications 👍 
- [ ] I noticed that your website depends a lot on the height of the screen which would create issues on some screens. Kindly fix these issues 👍 

Animation

Animation

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by V-Blaze** [comment]
I have gone through the comments in the review and I have made the necessary corrections to the code and the design.

**Review by OLIPLICHE — APPROVED** [review]
Hi @V-Blaze, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Implemented the changes requested by the previous review👏👏👏
- Linter is green (the last commit has a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**